### DISCUSSION TITLE
Can we run all spec files and suite with different mochaoptins in a single command with parallel execution?

### DISCUSSION THREAD

**Discussion by vijaynagr** [webdriverio/webdriverio#9047] [Q&A]
Hi,

I am wondering if there is a way to combine the below 2 lines and execute it in one line and the tests run on different chrome instances.

//Run all the spec files with a specific tags
./node_modules/.bin/wdio config/wdio.standalone.conf.js --mochaOpts.grep "@Drop1|@Drop2|@Drop3"

//Run 2 different suits with another set of mocha options
./node_modules/.bin/wdio config/wdio.standalone.conf.js --suite suite1 --suite suite2 --mochaOpts.grep "@Drop4"

Can these 2 different execution command be combined into one? Thanks in advance for helping out.

**Comment by christian-bromann** [comment]
How should this look like? How should a user pass in paramaters to make this work?

**Reply by vijaynagr** [reply]
I was hopping it should look something like below
./node_modules/.bin/wdio config/wdio.standalone.conf.js --mochaOpts.grep "@drop1|@Drop2|@drop3" --suite suite1 --suite suite2 --mochaOpts.grep "@Drop4"

I was hoping with the above command, it will run all the spec files will run in parallel and also the suits, suite1 and suite2 will also be run in parallel

but this is not working the way I was imagining.

**Reply by christian-bromann** [reply]
This can't work, how is WebdriverIO suppose to know which grep belongs to which suite parameter. There are no such connection. I would try to create a single grep.

**Reply by vijaynagr** [reply]
Thanks for clarifying
